code/modules/surgery/organs/organ_icon.dm

```diff
@@ -162,10 +162,13 @@ GLOBAL_LIST_EMPTY(limb_icon_cache)
 		new_icon_state = "[icon_name][gendered_icon ? "_f" : ""]"
 	else
 		if(gendered_icon)
-			if(dna.GetUIState(DNA_UI_GENDER))
-				gender = "f"
-			else
-				gender = "m"
+			switch(dna.GetUITriState(DNA_UI_GENDER))
+				if(0)
+					gender ="f"
+				if(1)
+					gender ="m"
+				else
+					gender = pick("f", "m")
```

Comment:
Same as above

Reply:
We don't have Genderless/Plural sprites. I'll default this to one value.